blueprints/addon/files/CONTRIBUTING.md

```diff
@@ -10,7 +10,7 @@
 
 * `<% if (yarn) { %>yarn lint:hbs<% } else { %>npm run lint:hbs<% } %>`
 * `<% if (yarn) { %>yarn lint:js<% } else { %>npm run lint:js<% } %>`
-* `<% if (yarn) { %>yarn lint:js --fix<% } else { %>npm run lint:js -- --fix<% } %>`
+* `<% if (yarn) { %>yarn lint:js:fix<% } else { %>npm run lint:js:fix<% } %>`
```

Comment:
Let's suggest `lint` and `lint:fix` isntead of all the individual `lint:xxxx` type scripts.